### ISSUE TITLE
OpenSpades 0.1.3 It closes alone

### ISSUE THREAD

**Issue by GhostDark123** [yvt/openspades#880]
Hi,

Recently I just downloaded OpenSpades Windows and when I try to open it automatically it closes alone, I want to see if that problem can be solved

**Comment by feikname** [comment]
What OS? No error message at all?

**Comment by GhostDark123** [comment]
No, I don't get any error, it just opens and closes by itself. my operating system is windows 7

**Comment by GhostDark123** [comment]
> What OS? No error message at all?

I just opened the OpenSpades in a .zip file and I get the error of ms-win-crt-runtime-l1-1-0.dll is missing on the computer

**Comment by GhostDark123** [comment]
ya instale el update de windows pero todavia sigue cerrandose

**Comment by JohnDowson** [comment]
Don't run it from a zip file, you need to extract it somewhere first.

**Comment by GhostDark123** [comment]
If I run it outside the .zip file when I run it it just closes
